### PR TITLE
build(webpack): Track webpack asset sizes

### PR DESCRIPTION
**Pull request by billyvg**
Some might say this would be abusing Sentry APM...

..... and they would be right

### CODE REVIEW

**Comment by billyvg** [comment]
hi @ckj

**Review by ckj — APPROVED** [review]

File: build-utils/sentry-instrumentation.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+class SentryInstrumentation {
+  constructor() {
+    // Only run if SENTRY_INSTRUMENTATION` is set or when in travis, only in the javascript suite
+    if (process.env.SENTRY_INSTRUMENTATION || process.env.TEST_SUITE === 'js') {
+      this.Sentry = require('@sentry/node');
+      require('@sentry/apm');
+
+      this.Sentry.init({
+        dsn: 'https://3d282d186d924374800aa47006227ce9@sentry.io/2053674',
```
Comment (billyvg) [inline build-utils/sentry-instrumentation.js]: Hard coding this because I want to eventually run this for local development.

**Review by billyvg — COMMENTED** [review]
This is something maybe silly we (@ckj) wanted to simply track asset sizes over time. There's obviously a lot of limitations at play here but there's low costs involved imo. I may in the future add tracking build durations (for travis and local dev).

**Review by evanpurkhiser — APPROVED** [review]
this is neat

**Pull request merged**